index.bs

```diff
@@ -391,6 +470,76 @@ interface mixin WebTransport {
         {{WebTransportCloseInfo/errorCode}} and its reason value set to the
         value of {{WebTransportCloseInfo/reason}}.
 
+: <dfn for="WebTransport" method>getStats()</dfn>
+:: Gathers stats for this {{WebTransport}}'s QUIC
+   connection and reports the result asynchronously.</p>
+
+   When close is called, the user agent MUST run the following steps:
+     1. Let |transport| be the WebTransport on which `getStats` is invoked.
+     1. Let |p| be a new promise.
+     1. Return |p| and continue the following steps in background.
+         1. Gather the stats from the underlying QUIC connection.
```

Comment:
Is this safe from a security and privacy standpoint with an HTTP connection?

Reply:
Some of them are safe, others are not.  I made it an error for now, and added a note saying we should revisit this and split them into "available only in dedicated" and "always available".

Reply:
Thanks.